front-end/e-learning/src/components/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
```

Comment:
put space to group imports.
One group for react related packages
another group for MUI pacakges
another group for custom made components

```suggestion
import React from 'react';

import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import Link from '@mui/material/Link';
import Typography from '@mui/material/Typography';
```